### DISCUSSION TITLE
spelling mistakes?

### DISCUSSION THREAD

**Discussion by lengyibai** [vuejs/pinia#1677] [Help and Questions]
![image](https://user-images.githubusercontent.com/74140255/191819007-0c5d5a63-05ca-47b9-98e9-36871da02c48.png)

pania => pinia

**Comment by mlzzen** [comment]
这是官方文档吗？我怎么看到pinia只有英文文档。

**Comment by lengyibai** [comment]
https://pinia.web3doc.top




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "MinLiang ***@***.***&gt;; 
发送时间: 2022年9月26日(星期一) 上午9:38
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [vuejs/pinia] spelling mistakes? (Discussion #1677)





 
这是官方文档吗？我怎么看到pinia只有英文文档。

**Reply by mlzzen** [reply]
你这是盗版网站啊
